pkgs/by-name/xe/xen_4_19/package.nix

```diff
@@ -55,4 +41,14 @@ callPackage (import ../generic/default.nix {
       patches = [ ];
     };
   };
-}) ({ ocamlPackages = ocaml-ng.ocamlPackages_4_14; } // genericDefinition)
+  descriptions = {
+    description = "Standard";
+    longDescription = ''
+      Standard version of the Xen Project Hypervisor. Uses forks of QEMU, SeaBIOS,
+      OVMF and iPXE provided by the Xen Project. This provides the vanilla Xen
+      experience, but wastes space and build time. A typical NixOS setup that runs
+      lots of VMs will usually need to build two different versions of QEMU when using
+      this Xen derivation (one fork and upstream).
+    '';
```

Comment:
Why does this need to be duplicated with the builder?

Reply:
It... doesn't. I forgot to remove it :P